samples/v1/language_entities_gcs.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "language_entities_gcs")
+
+// sample-metadata:
+//   title: Analyzing Entities (GCS)
+//   description: Analyzing Entities in text file stored in Cloud Storage
+//   usage: node samples/v1/language_entities_gcs.js [--gcs_content_uri "gs://cloud-samples-data/language/entity.txt"]
+
+'use strict';
+
+// [START language_entities_gcs]
+
+const {LanguageServiceClient} = require('@google-cloud/language').v1;
+
+/**
+ * Analyzing Entities in text file stored in Cloud Storage
+ *
+ * @param gcsContentUri {string} Google Cloud Storage URI where the file content is located.
+ * e.g. gs://[Your Bucket]/[Path to File]
+ */
+function sampleAnalyzeEntities(gcsContentUri) {
+  const client = new LanguageServiceClient();
+  // const gcsContentUri = 'gs://cloud-samples-data/language/entity.txt';
+
+  // Available types: PLAIN_TEXT, HTML
+  const type = 'PLAIN_TEXT';
+
+  // Optional. If not specified, the language is automatically detected.
+  // For list of supported languages:
+  // https://cloud.google.com/natural-language/docs/languages
+  const language = 'en';
+  const document = {
+    gcsContentUri: gcsContentUri,
+    type: type,
+    language: language,
+  };
+
+  // Available values: NONE, UTF8, UTF16, UTF32
+  const encodingType = 'UTF8';
+  const request = {
+    document: document,
+    encodingType: encodingType,
+  };
+  client
+    .analyzeEntities(request)
+    .then(responses => {
```

Comment:
similar comment to @JustinBeckwith, I think we should standardize on `async`/`await`.